buildSrc/src/main/java/org/elasticsearch/gradle/BwcVersions.java

```diff
@@ -345,7 +349,14 @@ public void compareToAuthoritative(List<Version> authoritativeReleasedVersions)
         wireCompat.addAll(groupByMajor.get(currentVersion.getMajor()));
         wireCompat.sort(Version::compareTo);
 
-        return unmodifiableList(wireCompat);
+        return Architecture.current() == Architecture.AARCH64
+            ? unmodifiableList(filterAarch64SupportedVersions(wireCompat))
+            : unmodifiableList(wireCompat);
+    }
+
+    @NotNull
```

Comment:
Are we doing this consistently? If not we might as well not bother.

Reply:
removed. brought to you by idea